lightspark-sdk/src/commonMain/kotlin/com/lightspark/sdk/graphql/CreateUmaInvoice.kt

```diff
@@ -8,8 +8,17 @@ const val CreateUmaInvoiceMutation = """
     ${'$'}amountMsats: Long!
     ${'$'}metadataHash: String!
     ${'$'}expirySecs: Int = null
+    ${'$'}receiverHash: String
```

Comment:
Same here

Reply:
it seems like not needed?? for example https://github.com/lightsparkdev/kotlin-sdk/pull/174/files#diff-a3974086a91f49faa928b13e5e700377265b59a374ebf9e127ccb428e27e55f1R11 but updated anyways